src/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -30,9 +30,6 @@
   <ItemGroup Condition="'$(TargetsNETCoreApp)'=='true'">
     <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.netcoreapp.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.3'">
-    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.netstandard1.3.cs" />
-  </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netstandard'">
```

Comment:
nit: above we are using TargetsNETCoreApp, should we use TargetNetStandard here too instead?

Reply:
The reason why we're using TargetsNETCoreApp is because it is used by both, netcoreapp2.1 and netcoreapp targetgroups, but here it is specific to netstandard. Since it is the only version of netstandard used I think it is better to leave it like is.